src/SonarQube.TeamBuild.Integration/TfsCoverageReportLocator.cs

```diff
@@ -18,43 +18,32 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using System;
 using SonarQube.Common;
-using SonarQube.TeamBuild.Integration.Interfaces;
 
 namespace SonarQube.TeamBuild.Integration
 {
-    public class BuildVNextCoverageReportProcessor : CoverageReportProcessorBase
+    public class TfsCoverageReportLocator : ICoverageReportLocator
     {
-        #region Public methods
+        private readonly ILogger logger;
 
-        public BuildVNextCoverageReportProcessor()
-            : this(new CoverageReportConverter())
+        public TfsCoverageReportLocator(ILogger logger)
         {
+            this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
         }
 
-        public BuildVNextCoverageReportProcessor(ICoverageReportConverter converter)
-            : base(converter)
+        public bool TryGetBinaryCoveragePath(AnalysisConfig config, ITeamBuildSettings settings, out string binaryCoveragePath)
         {
-        }
-
-        #endregion Public methods
-
-        #region Overrides
-
-        protected override bool TryGetBinaryReportFile(AnalysisConfig config, ITeamBuildSettings settings, ILogger logger, out string binaryFilePath)
-        {
-            binaryFilePath = TrxFileReader.LocateCodeCoverageFile(settings.BuildDirectory, logger);
+            binaryCoveragePath = TrxFileReader.LocateCodeCoverageFile(settings.BuildDirectory, logger);
```

Comment:
I think TrxFileReader could be merged with TfsCoverageReportLocator, did not want to write more tests now, though